# Simpson's Paradox
using `admission_data.csv` to this exercise

In [1]:
# Loading the datasets and the libraries 

import pandas as pd
import numpy as np

df = pd.read_csv('admission_data.csv')
df.head()

student_id  gender      major  admitted
0       35377  female  Chemistry     False
1       56105    male    Physics      True
2       31441  female  Chemistry     False
3       51765    male    Physics      True
4       53714  female    Physics      True

In [2]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 500 entries, 0 to 499
Data columns (total 4 columns):
student_id    500 non-null int64
gender        500 non-null object
major         500 non-null object
admitted      500 non-null bool
dtypes: bool(1), int64(1), object(2)
memory usage: 12.3+ KB


In [3]:
df['gender'].value_counts()

female    257
male      243
Name: gender, dtype: int64

In [4]:
df_male_total = df[(df.gender == 'male')]
df_male_total.head()

student_id gender    major  admitted
1       56105   male  Physics      True
3       51765   male  Physics      True
6       25946   male  Physics      True
8       55247   male  Physics     False
9       35838   male  Physics      True

In [10]:
df_male_ad = df_male_total[(df_male_total.admitted == True)]
df_male_ad.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 118 entries, 1 to 498
Data columns (total 4 columns):
student_id    118 non-null int64
gender        118 non-null object
major         118 non-null object
admitted      118 non-null bool
dtypes: bool(1), int64(1), object(2)
memory usage: 3.8+ KB


In [7]:
df_female_total = df[(df.gender == 'female')]
df_female_total.head()

student_id  gender      major  admitted
0       35377  female  Chemistry     False
2       31441  female  Chemistry     False
4       53714  female    Physics      True
5       50693  female  Chemistry     False
7       27648  female  Chemistry      True

In [9]:
df_female_ad = df_female_total[(df_female_total.admitted == True)]
df_female_ad.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 74 entries, 4 to 497
Data columns (total 4 columns):
student_id    74 non-null int64
gender        74 non-null object
major         74 non-null object
admitted      74 non-null bool
dtypes: bool(1), int64(1), object(2)
memory usage: 2.4+ KB


In [6]:
df_male_total.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 243 entries, 1 to 499
Data columns (total 4 columns):
student_id    243 non-null int64
gender        243 non-null object
major         243 non-null object
admitted      243 non-null bool
dtypes: bool(1), int64(1), object(2)
memory usage: 7.8+ KB


### Proporção e taxa de admissão para cada sexo

<center> <b>Male</b></center>

| Total_male|Admitted_male|% male|Total_female|Admitted_female|% female|
|------|------|------|------|------|------|
|  243  | 118|48.55%| 257 | 74|28.79%|



### Proporção e taxa de admissão para graduandos de física de cada gênero